test/uk/gov/hmrc/fileupload/controllers/RedirectionFeatureSpec.scala

```diff
@@ -33,7 +33,7 @@ class RedirectionFeatureSpec extends UnitSpec with ScalaFutures with TestApplica
   private val allowedHosts = Seq[String]("gov.uk","localhost")
```

Comment:
Should you write some more UT to be sure that this new redirect handling works as expected?

Reply:
Yeah, I don't need to wait for IT tests to write a unit test.